eventnotifier/slack/slack.go

```diff
@@ -241,7 +241,7 @@ func (n *Notifier) buildSlackMessage(obj client.Object, event corev1.Event, noti
 	}
 
 	if err != nil {
-		logger.Errorw("unable to retrieve user info", "error", err)
```

Comment:
IMO it's a warning because without the user info we are not able to send the notification the user 😕

Reply:
Yes, this was a hard choice. It's definitely a warning if it happens when we don't expect, but we do expect it at least once (so far). Maybe I can find a way not to send any events until we have the UserInfo